hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -509,6 +509,256 @@ def check_core_dumps(reason=''):
     return str(hard_core_dump_value)
 
 
+def check_service_status(service_name, state):
+    '''
+    Ensure that the given service is not running on the system. Return False if the service is running.
```

Comment:
I think we should take status input as running or disabled and return True/False accordingly.